src/Components/Link/component.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+
+type LinkProps = {
+    label: string,
+    linkTo: string
+};
+
+type LinkState = {
+    target: string
+};
+
+class Links extends Component<LinkProps, LinkState> {
+    state = {
+        target: '_self'
+    };
+
+    componentWillMount() {
+        if (this.props.target) {
+            this.setState({
+                target: '_blank'
+            });
+        }
+    }
+
+    render() {
+        return (
+            <a target={this.state.target} href={this.props.linkTo}>
```

Comment:
How about {...props} instead of each prop `target` and `href`? There are more attributes to `a` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Attributes